code/controllers/subsystem/processing/fluids.dm

```diff
@@ -3,3 +3,22 @@ PROCESSING_SUBSYSTEM_DEF(fluids)
 	wait = 20
 	stat_tag = "FD" //its actually Fluid Ducts
 	flags = SS_NO_INIT | SS_TICKER
+
+	///We randomly pick a duct from this list to start building a network
+	var/list/prime_queued = list()
```

Comment:
Why pick from this list? what happens if there's more then one "line" that wants to build at once?

Reply:
We only ever let one line build at once. This is the list of ducts that can start building, and we only ever start building of no other ducts are being built